pkg/network/ovn_kubernetes.go

```diff
@@ -624,9 +625,17 @@ func getDisableUDPAggregation(cl crclient.Reader) bool {
 	disableUDPAggregation := cm.Data["disable-udp-aggregation"]
 	if disableUDPAggregation == "true" {
 		return true
+	} else if disableUDPAggregation == "false" {
+		return false
 	} else {
 		klog.Warningf("Ignoring unexpected udp-aggregation-config override value disable-udp-aggregation=%q", disableUDPAggregation)
 	}
+
+	// Disable on s390x because it sometimes doesn't work there; see OCPBUGS-2532
+	if goruntime.GOARCH == "s390x" {
```

Comment:
For other reviewers, I _think_ this `GOARCH` value is from here: https://github.com/golang/go/blob/master/src/go/build/syslist.go